vignettes/gallery.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Gallery"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Gallery}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+<br><br>
+
+```{r}
+#| echo: false
+#| eval: !expr rlang::is_installed(c("commonmark", "DT", "tibble"))
+# Contributors: to add your pal to this table, add one row per pal in the
+# following `tribble()` call. If you've authored a pal extension that includes
+# multiple pals, include a row for each pal.
+gallery <- 
+  tibble::tribble(
+    ~description, ~role, ~interface,  ~type, 
```

Comment:
Is this all relevant information to include? Am I missing anything key?

Ideas I had tossed around were 1) automatic generation of the `prompt_new()` code, 2) author, or 3) tags.

Reply:
I don't find the `type` column all that informative. What I do think would be very good to have is a column with "Use it via" information, so a minimal piece of code that you can c+p into your console and get going. For markdown files like mine, it would be `prompt_new()` but with the right URL (which needs to be the raw md content, not the github rendered page). For an extension package, it would be `install.packages()` or `pak::pak()`, with right name or user/repo info. Given that the URL for my markdown thing is rather long: a copy to clipboard button would be sooo nice, especially if that allowed you to hide the long URL somehow.

I don't think you need tags yet, I would revisit that once you feel you need it. 

Re author: you/me as a user can live without it but since you would like to give visibility and encourage participation, I would add it.